sidebars.js

```diff
@@ -261,7 +261,18 @@ const sidebars = {
             'en/integrations/data-visualization/deepnote',
             'en/integrations/data-visualization/draxlr-and-clickhouse',
             'en/integrations/data-visualization/explo-and-clickhouse',
-            'en/integrations/data-visualization/grafana-and-clickhouse',
+            {
```

Comment:
The items are in the alphabetical order, but I'd argue some of them are more important and should be at the top
- Grafana
- Metabase
- Superset
- Looker
- PowerBI
- Tableau
- Quicksight
...the rest

Reply:
Considering the length of the list it might be best to keep it alphabetical, although I do agree that some of these connectors are more popular than others.

Also consider that this list is already hidden under **Clients and Drivers -> Business Intelligence**. Most people will find these pages via search (either Google, or the doc's search) instead of manually digging through the sidebar